SampleApp/SampleApp/Delegates/SceneDelegate.swift

```diff
@@ -18,6 +19,9 @@ class SceneDelegate: UIResponder, UIWindowSceneDelegate {
         willConnectTo session: UISceneSession,
         options connectionOptions: UIScene.ConnectionOptions
     ) {
+        if let userInfo = connectionOptions.notificationResponse?.notification.request.content.userInfo as? [String: Any] {
```

Comment:
Ah yes, this can fire as well when userNotificationCenter 's `didReceive` gets called.
I was implementing this for when app is inactive(force-quit), however, it looks like userNotificationCenter `didReceive` can do the same job. So I'll remove this too. Thank you!

Reply:
Did you do the same on Frontier?

Reply:
Yes, I remove it from Frontier too.